lms/static/coffee/src/instructor_dashboard/membership.coffee

```diff
@@ -670,6 +670,569 @@ class AuthList
       success: (data) -> cb?(data)
       error: std_ajax_err => @$request_response_error.text gettext "Error changing user's permissions."
 
+class EmailSelectors
+  constructor: (@$container, @$section, params={}) ->
+    params = _.defaults params,
+      label : $container.data 'label'
+
+    template_html = $("#email-list-widget-template").html()
+    @$container.html Mustache.render template_html, params
+    @cur_column = 0
+    @$table = $("#queryTableBody")
+    @labelArray = ($container.data 'selections').split '<>'
+    @$list_selector = @$container.find 'select.single-email-selector'
+    # populate selectors
+    @$list_selector.empty()
+    @$list_endpoint = $container.data 'list-endpoint'
+    @$rolename = $container.data 'rolename'
+    @$list_selector.append $ '<option/>'
+    if this.$container.attr('data-label') =='Select Section'
+      @load_list()
+    else if this.$container.attr('data-label') =='Select Problem'
```

Comment:
Throughout this Coffeescript, many of the operators lack surrounding whitespace. This makes the code much more difficult to read (Coffeescript is bad enough :P).

Operators should _always_ be surrounded by whitespace, except when specifying default params in a function definition.

Please add whitespace around all operators in these changes.